contents/node/nodejs.md

```diff
@@ -181,14 +181,14 @@ Apart from acting as <tooltip content="See here for some common use cases of npm
 
 Node's module system allows you to include other JavaScript files and thus makes it easy to reuse external libraries and organize your code into separate parts with limited responsibilities.
 
-Node comes bundled with useful [core modules](https://nodejs.org/api/) such as the `fs` (file system) module which includes classes, methods and events to deal with file I/O operations and the `http` module which helps Node to transfer data over HTTP.
+Node comes bundled with useful [core modules](https://nodejs.org/api/) such as the `fs` (file system) module which includes classes, methods and events to deal with file I/O operations and the `https` module which helps Node to transfer data over HTTP.
 There are also many useful and well-tested modules maintained by the community and external developers such as <tooltip content="A library providing various useful wrapper functions over JavaScript's promises">*Bluebird*</tooltip> on <tooltip content="the largest ecosystem of open source libraries in the world!">npm</tooltip>.
 
 This makes development a breeze -- if a specific functionality has a large development overhead, you could look to such modules to speed things up.
 
 Importing modules is easy - simply use the `require()` function and provide the module identifier or the file path.
 ```js
-const http = require('http'); // import a core module
+const https = require('https'); // import a core module
```

Comment:
Is this correct?

Reply:
Opps, this is a mistake, but it is still a correct syntax as there is a `https` nodejs module that does the same thing. Will correct this

Reply:
done.